lib/log_helper.py

```diff
@@ -20,21 +20,34 @@
 
 
 class LogHelper(object):
-    def __init__(self, logfile=None, level=logging.DEBUG, verbose=False):
+    def __init__(self, logfile=None, verbose=False):
         self.logfile = logfile
-        self.loglevel = level
 
         self._directory_setup()
 
         logger = logging.getLogger()
-        logging.basicConfig(filename=logfile, level=level)
-        print("Logs available at %s" % logfile)
+        logger.setLevel(logging.DEBUG)
+
+        # stdout - INFO
+        sh = logging.StreamHandler(sys.stdout)
+        sh.setLevel(logging.INFO)
+        sh.setFormatter(logging.Formatter('%(message)s'))
+        logger.addHandler(sh)
+
+        logger.info("Logs available at %s" % logfile)
+
         # NOTE(maurosr): RotatingFileHandler expects file size in bytes, in
         # short terms we're defining 2MB limit here.
-        logger.addHandler(logging.handlers.RotatingFileHandler(
-            logfile, maxBytes=2 << 20, backupCount=1))
+        rfh = logging.handlers.RotatingFileHandler(logfile, maxBytes=2 << 20,
```

Comment:
That would be nice. Should we add TODO or IMPROVEMENT tags for this kind of stuff?